MODULE.bazel

```diff
@@ -65,3 +65,26 @@ use_repo(llvm, "llvm_toolchain")
 # use_repo(llvm, "llvm_toolchain_llvm") # if you depend on specific tools in scripts
 
 register_toolchains("@llvm_toolchain//:all")
+
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+name = "whereami",
+    build_file_content = """
+cc_library(
+    name = "whereami",
+    srcs = [
+        "whereami.c", "whereami.h",
+    ],
+    hdrs = [
+        "whereami.h",
+    ],
+    copts= ["-I./external/_main~_repo_rules~whereami/"],
```

Comment:
Don't do this. Rather do includes = [ "." ].

Reply:
Didn't work:

`` `

Reply:
I don't particularly want to patch up the library.